include/unk_02031734.h

```diff
@@ -0,0 +1,215 @@
+#ifndef POKEDIAMOND_UNK_02031734_H
+#define POKEDIAMOND_UNK_02031734_H
+
+#include "global.h"
```

Comment:
only the essentials for the header file should be in this include list, the rest should be in the .c